test/example.test.js

```diff
@@ -1,8 +1,21 @@
 // IMPORT MODULES under test here:
-// import { example } from '../example.js';
+
+import { createCountString } from "../utils.js";
+
 
 const test = QUnit.test;
 
+
+test('Should verify that function creates string with state values', (expect) => {
+    let expected = "You have changed Cities 4 times, visited the water 2 times, and moved 0 times";
+    let actual = createCountString(4, 2, 0);
+    expect.equal(expected, actual, "Should produce a string with the number 4, 2, and 0 respectively within")
```

Comment:
These eslint rules may seem annoying but it is also good practice to check them before merging a PR.  Think of eslint as the senior dev that is reviewing your code.  Also if your build fails for a linting error, github never runs your tests, which is important for working on larger projects.  So make sure and check the details of the red x on your PR's, usually it is a simple fix!

Reply:
I see what you're talking about. I'll have it fixed by the next push. It's the double quotes on my strings. Eslint wants single quotes